action.yml

```diff
@@ -1,10 +1,10 @@
 name: 'Checkout'
-description: 'Checkout a Git repository.'
+description: 'Clone and checkout a Git repository at a particular version'

```

Comment:
By default, we checkout what we think is the right sha for the user build (a merge commit on pr's, the head of a the branch on pushes, ect). I think this is misleading and we should just say

`Clone and checkout a git repository.`

Reply:
@ethomson i pushed changes to master for checkout v2 (beta right now). It doesnt do a full fetch anymore, now it just fetches the single commit (improved perf for mainline scenario)

Curious if you think your proposed description still fits? I'll defer to your judgement

Reply:
I think that just saying "checkout a git repository" - without any additional information 
- suggests to the reader that it will be at the remote HEAD.  Indicating that it will be "at a particular version" hints that a) it might not be the repository's default and b) it might be able to be overridden.

Not tied to this verbiage.

Reply:
Thanks makes sense. It was the "Clone" part of your proposed changes that I was worried about. The phrase "at a particular version" is good. Let me know if you want to update the PR otherwise i can make the changes. There is an `npm run gendocs` command that will regen the input descriptions in the README.md from the action.yml

Reply:
Cool.  I pushed an update with some minor tweaks.  TBQH the descriptions are already good so this is super minor.